internal/cmd/cli/help/help_setup.go

```
@@ -0,0 +1,114 @@
+		var buffer bytes.Buffer
+		err := engine.Execute(&buffer, "command_help.md", c)
+		if err != nil {
+			c.PrintErrln("Error executing help template:", err)
+			return
+		}
+		rendered, err := renderer.Render(buffer.String())
+		if err != nil {
+			c.Print(buffer.String())
+			return
+		}
+		lipgloss.Fprint(c.OutOrStdout(), rendered)
```

golangci-lint (2.12.2)

[error] Error return value of `lipgloss.Fprint` is not checked

(errcheck)